understanding/resize-text.html

```diff
@@ -40,7 +40,7 @@ <h2>意図</h2>
             <div class="note"><div role="heading" class="note-title marker" aria-level="3">訳注</div><p>ほとんどのモダンブラウザでは拡大機能が提供されている。また、IE6 は、サポートが終了したブラウザである。<a href="http://www.atmarkit.co.jp/ait/articles/1503/11/news134.html">いよいよ完全終了へ。Internet Explorer（IE）サポート終了スケジュール：Tech TIPS - ＠IT</a> も参照されたい。</p></div>
             <p>コンテンツ制作者がユーザエージェントによるズームサポートの提供がない技術を使用している場合、コンテンツ制作者は、このタイプの機能を直接提供する、又はユーザエージェントによって提供されるこのタイプの機能と連携するコンテンツを提供する責任がある。ユーザエージェントがズーム機能を提供していないが、利用者にテキストの大きさの変更するのを許可する場合、コンテンツ制作者は、テキストの大きさを変更するときにコンテンツが利用可能なままであることを保証する責任がある。</p>
             <p>ラベルとして機能し、コンテンツにアクセスするために利用者によるアクティベーションを必要とするユーザインタフェースコンポーネントの中には、ラベルコンテンツを収容するのに十分な幅がないものがある。例えば、ウェブメールアプリケーションにおける件名のカラムは、全ての可能性がある件名の見出しを収容するために十分な幅でなくてもよいが、件名の見出しをアクティブにすることで、利用者は、完全な件名の見出しを伴う完全なメッセージを得る。ウェブベースのスプレッドシートでは、カラム内に表示するには長すぎるセルのコンテンツは切り落とされ、そのセルの全コンテンツは、セルがフォーカスを受け取ったとき、利用者に提供される。ユーザインタフェースコンポーネントのコンテンツは、利用者がカラムの幅の大きさを変更することができるユーザインタフェースでも同様に、広くするかもしれない。このタイプのユーザインタフェースコンポーネントでは、改行が必須ではない。コンポーネントの完全なコンテンツが、フォーカスを置く又は利用者のアクティベーションの後に利用可能であり、この情報にアクセスすることができることが示され、それが切り捨てられるという事実の他にいずれかの方法で利用者へ提供される場合、切り捨ては許容される。</p>
-            <p>コンテンツが 200％ まで、言い換えれば、幅と高さが 2 倍になるまで拡大可能である場合、コンテンツは達成基準を満たしている。コンテンツ制作者はそれ以上の拡大をサポートしてもよいが、拡大が極端に大きくなると、アダプティブレイアウトはユーザビリティの問題を引き起こす可能性がある。例えば、語句の広くなりすぎると、使用可能な横幅に収まりきらず、切り取られる可能性がある。レイアウト上の制約によって、拡大していったときに、テキストが他のコンテンツと重なり合ってしまうこともある。あるいは、文章中の一つの単語だけが各行にある状態になると、その文章が縦に表示されてしまって読みづらくなってしまう。</p>
+            <p>コンテンツが 200％ まで、言い換えれば、幅と高さが 2 倍になるまで拡大可能である場合、コンテンツは達成基準を満たしている。コンテンツ制作者はそれ以上の拡大をサポートしてもよいが、拡大が極端に大きくなると、アダプティブレイアウトはユーザビリティの問題を引き起こす可能性がある。例えば、語句自体の幅が広くなりすぎると、使用可能な横幅に収まりきらず、切り取られる可能性がある。レイアウト上の制約によって、拡大していったときに、テキストが他のコンテンツと重なり合ってしまうこともある。あるいは、文章中の一つの単語だけが各行にある状態になると、その文章が縦に表示されてしまって読みづらくなってしまう。</p>
```

Comment:
原文これですね。

For example, words may be too wide to fit into the horizontal space available to them, causing them to be truncated; layout constraints may cause text to overlap with other content when it is scaled larger; or only one word of a sentence may fit on each line, causing the sentence to be displayed as a vertical column of text that is difficult to read.

too wide to fit into は「広くなりすぎると収まりきらない」というより「収まりきらないほど広くなる」ですかね?
mayを「可能性がある」としていますが、列挙されている次の文では may を「こともある」としているので、揃えたい感じがします (今の翻訳文からはわかりにくいですが、for exampleの内容がセミコロン区切りで3つ列挙されているという構造)。たとえばこう?

```suggestion
            <p>コンテンツが 200％ まで、言い換えれば、幅と高さが 2 倍になるまで拡大可能である場合、コンテンツは達成基準を満たしている。コンテンツ制作者はそれ以上の拡大をサポートしてもよいが、拡大が極端に大きくなると、アダプティブレイアウトはユーザビリティの問題を引き起こす可能性がある。例えば、語句自体の幅が使用可能な横幅に収まりきらないほど広くなり、切り取られることがある。レイアウト上の制約によって、拡大していったときに、テキストが他のコンテンツと重なり合ってしまうこともある。あるいは、文章中の一つの単語だけが各行にある状態になり、その文章が縦に表示されてしまって読みづらくなってしまうこともある。</p>
```

Reply:
@bakera 

> 語句自体の幅が使用可能な横幅に収まりきらないほど広くなり、切り取られることがある。

日本語としては、
語句自体の幅が使用可能な横幅に収まりきらないほど広いために、切り取られることがある。

のほうが自然でしょうか？

Reply:
前の文で「拡大が極端に大きくなると」と言っているので、「(拡大が極端に大きくなると)広くなり」ということで良いのではないかと思っています。どうでしょう?

Reply:
@bakera 

> 前の文で「拡大が極端に大きくなると」と言っているので、「(拡大が極端に大きくなると)広くなり」ということで良いのではないかと思っています。どうでしょう?

なるほど、通しで読むとそこまで違和感ないですね。よいと思います！